crates/holochain_p2p/src/types/actor.rs

```diff
@@ -362,7 +364,7 @@ ghost_actor::ghost_chan! {
 
         /// Dump network metrics.
         fn dump_network_metrics(
-            dna_hash: Option<DnaHash>,
+            dna_hash: MaybeDnaHash,
```

Comment:
i'd rather rename the attribute than making a new type, as the `Option..` already expresses the _maybe_. if in doubt i'd just leave the original code here.

Reply:
Alas, the macro was complaining without this alias. Rather than hunt down the reason, I decided to appease it with a newtype.